.env.example

```
@@ -1,8 +1,94 @@
+
+# Absolute path to the .elizadb file - not mandatory to define
+# PGLITE_DATA_DIR="your/absolute/path/to/eliza-plugin-hedera/.eliza/.elizadb"
+
+---------------------------------
+
+### HEDERA PLUGIN CONFIG ###
+# Required: Hedera account private key (accepts DER encoded keys - both ED25519 and ECDSA)
+HEDERA_PRIVATE_KEY=
+
+# Required: Hedera account ID (ex. `0.0.5393196`)
+HEDERA_ACCOUNT_ID=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)